docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -31,6 +31,420 @@ For more information about versions, build numbers, and Office Online Server, se
+||[items](/javascript/api/excel/excel.rangecollection#items)|Gets the loaded child items in this collection.|
+|[RangeFill](/javascript/api/excel/excel.rangefill)|[pattern](/javascript/api/excel/excel.rangefill#pattern)|Gets or sets the pattern of a Range. See Excel.FillPattern for details. LinearGradient and RectangularGradient are not supported.|
+||[patternColor](/javascript/api/excel/excel.rangefill#patterncolor)|Sets HTML color code representing the color of the Range pattern, of the form #RRGGBB (e.g. "FFA500") or as a named HTML color (e.g. "orange").|
+||[patternTintAndShade](/javascript/api/excel/excel.rangefill#patterntintandshade)|Returns or sets a double that lightens or darkens a pattern color for Range Fill, the value is between -1 (darkest) and 1 (brightest), with 0 for the original color.|
+||[tintAndShade](/javascript/api/excel/excel.rangefill#tintandshade)|Returns or sets a double that lightens or darkens a color for Range Fill, the value is between -1 (darkest) and 1 (brightest), with 0 for the original color.|
+|[RangeFont](/javascript/api/excel/excel.rangefont)|[strikethrough](/javascript/api/excel/excel.rangefont#strikethrough)|Represents the strikethrough status of font. A null value indicates that the entire range doesn't have uniform Strikethrough setting.|
+||[subscript](/javascript/api/excel/excel.rangefont#subscript)|Represents the Subscript status of font.|
+||[superscript](/javascript/api/excel/excel.rangefont#superscript)|Represents the Superscript status of font.|
+||[tintAndShade](/javascript/api/excel/excel.rangefont#tintandshade)|Returns or sets a double that lightens or darkens a color for Range Font, the value is between -1 (darkest) and 1 (brightest), with 0 for the original color.|
+|[RangeFormat](/javascript/api/excel/excel.rangeformat)|[autoIndent](/javascript/api/excel/excel.rangeformat#autoindent)|Indicates if text is automatically indented when text alignment is set to equal distribution.|
+||[indentLevel](/javascript/api/excel/excel.rangeformat#indentlevel)|An integer from 0 to 250 that indicates the indent level.|
+||[readingOrder](/javascript/api/excel/excel.rangeformat#readingorder)|The reading order for the range.|
+||[shrinkToFit](/javascript/api/excel/excel.rangeformat#shrinktofit)|Indicates if text automatically shrinks to fit in the available column width.|
+|[RemoveDuplicatesResult](/javascript/api/excel/excel.removeduplicatesresult)|[removed](/javascript/api/excel/excel.removeduplicatesresult#removed)|Number of duplicated rows removed by the operation.|
+||[uniqueRemaining](/javascript/api/excel/excel.removeduplicatesresult#uniqueremaining)|Number of remaining unique rows present in the resulting range.|
+|[ReplaceCriteria](/javascript/api/excel/excel.replacecriteria)|[completeMatch?](/javascript/api/excel/excel.replacecriteria#completematch)|Specifies whether the match needs to be complete or partial. Default is false (partial).|
```

Comment:
Shouldn't the script be stripping the question mark from the end of this property name? (same issue exists with the next few properties in this table, and also a few other places within the table)  Sorry I didn't catch this earlier when reviewing the script PR.

Reply:
@AlexJerabek good work fixing this; marking this conversation as resolved.